OpenRA.Mods.Common/UpdateRules/Rules/20180923/FixedHeliRTBTurnToLand.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class FixedHeliRTBTurnToLand : UpdateRule
+	{
+		public override string Name { get { return "VTOLs no longer ignore TurnToLand when returning to base"; } }
+		public override string Description
+		{
+			get
+			{
+				return "Previously, VTOLs would always turn to their InitialFacing when landing on a resupplier.\n" +
+					"Now 'TurnToLand: false' (which is the internal default) will actually prevent them from doing so.";
+			}
+		}
+
+		List<string> locations = new List<string>();
+
+		public override IEnumerable<string> AfterUpdate(ModData modData)
+		{
+			if (locations.Count > 0)
+				yield return "Check the following actors if they need 'TurnToLand: true'\n" +
+					"in order to maintain the desired landing behavior:\n" +
+					UpdateUtils.FormatMessageList(locations);
+
+			locations.Clear();
+		}
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			foreach (var aircraftNode in actorNode.ChildrenMatching("Aircraft"))
+			{
+				var vtolNode = aircraftNode.LastChildMatching("VTOL");
+				if (vtolNode == null)
+					yield break;
+
+				var isVTOL = FieldLoader.GetValue<bool>("VTOL", vtolNode.Value.Value);
+				if (!isVTOL)
+					yield break;
+
+				var turnToLandNode = aircraftNode.LastChildMatching("TurnToLand");
+				if (turnToLandNode == null)
+				{
+					locations.Add("{0} ({1})".F(actorNode.Key, actorNode.Location.Filename));
+					yield break;
```

Comment:
We only report the actor and file anyway, so this should be fine (and I think we don't really support multiple Aircraft traits yet, I just used a foreach to be on the safe side and because @pchote said he'd generally prefer ChildrenMatching over LastChildMatching for updating traits).